doc/spec.md

```diff
@@ -3826,6 +3911,21 @@ x                                       # set([1, 3])
 x.remove(2)                             # error: element not found
 ```
 
+<a id='set·symmetric_difference'></a>
+### set·symmetric_difference
+
+`S.symmetric_difference(y)` returns a new set which includes all items in S and y, except for items which are in both.
```

Comment:
Let's state the insertion order: the x-only elements first followed by the y-only elements. Ditto for the other operators.